definitions.json

```diff
@@ -51,5 +51,6 @@
     "the talos principle": "a benchmark for Vulkan",
     "wat": "wat 1.0 (GNU Edition) Usage: !wat [word]     Display the meaning of 'word'",
     "normie": "If you have any kind of social skills then you're a 'normie'",
-    "venn": "A type of diagram"
+    "venn": "A type of diagram",
+    "pc": "https://cdn.discordapp.com/attachments/270406768750886912/493342479962210307/8195cce.png"
```

Comment:
don't use wat to post images

Reply:
What do you suggest then as an alternative?